docs/about.rst

```diff
@@ -0,0 +1,27 @@
+About Read the Docs
+===================
+
+Read the Docs is a C Corporation registered in Oregon.
+Our bootstrapped company is owned and fully controlled by the founders,
+and fully funded by our customers and advertisers.
+This allows us to **focus 100% on our users**. 
+
+We primarily make money in two ways:
+
+* |com_brand| - where we `provide <https://readthedocs.com>`__ a valuable paid service to companies.
+* |org_brand| - where we provide a free service to the open source community, funded via :doc:`/advertising/ethical-advertising`.
+
+We believe that having both paying customers and ethical advertising is the best way to create a sustainable platform for our users.
+We have built something that we expect to last a long time,
+and we are able to make decisions based on the best interest of our community and customers only.
+
+All of the source code for Read the Docs is open source.
+You are welcome to :doc:`contribute </contribute>` the features you want or run your own instance.
+We should note that we generally only support our hosted versions as a matter of :doc:`philosophy </open-source-philosophy>`.
```

Comment:
```suggestion
We should note that we generally only support our hosted versions as a matter of :doc:`our philosophy </open-source-philosophy>`.
```

Or "open source philosophy" maybe? This currently sounds like philosophy in general.